release-notes/release-notes56.md

```diff
@@ -69,6 +69,10 @@ For information on the `emojiimages` database and `emoticons_database` option, s
 
 For information on the `emoticons_images_url` option, see: [Emoticons plugin - `emoticons_images_url`]({{ site.baseurl }}/plugins/emoticons/#emoticons_images_url).
 
+### New Notification events
```

Comment:
I don't think we need 2 release notes for the new events. Could we merge them into a single "New editor events" note instead please?